src/index.html

```diff
@@ -6,13 +6,58 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img">
+        <img
+          class="card__item"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <h3 class="card__code">Product code: 195434</h3>
+
+      <div>
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </div>
+
+      <div class="card__price">Price:</div>
```

Comment:
The price should be wrapped in a `p` tag with a specific BEM class, and the actual price should be in another `p` tag with a different BEM class, according to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).